examples/PSQL/README.md

```diff
@@ -0,0 +1,6 @@
+# Example Postgres SQL configuration files
+This folder contains example of a database connection to Postgres Versions 8.2 - 9.4 which are not currently supported in ArcGIS insights-connector-guide/wiki/Create-a-connector-type-bundle
```

Comment:
This folder contains example configuration files to add a connector for PostgreSQL versions 8.2 - 9.4, which are not currently supported in ArcGIS insights-connector-guide/wiki/Create-a-connector-type-bundle

I'm not sure why that wiki page is mentioned. Did you mean to include it or just say that they aren't supported in Insights?